addons/portal/static/src/js/portal_wizard_user_field.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import { ListRenderer } from "@web/views/list/list_renderer";
+import { X2ManyField } from "@web/views/fields/x2many/x2many_field";
+import { registry } from "@web/core/registry";
+
+const { useSubEnv } = owl;
+
+export class PortalWizardUserListRenderer extends ListRenderer {
+    setup() {
+        super.setup();
+        this.isPortalActionOngoing = false;
+        const onClickViewButton = this.env.onClickViewButton;
+        useSubEnv({
+            onClickViewButton: async (params) => {
```

Comment:
I see stuff like ``async onClickViewButton(params)``, which one is the best way of writing :D ?